src/app/components/MediaLoader/configs/liveMedia.ts

```diff
@@ -7,7 +7,7 @@ export default ({
   basePlayerConfig,
 }: ConfigBuilderProps): ConfigBuilderReturnProps => {
   const { model: liveMediaBlock } = filterForBlockType(blocks, 'liveMedia');
-  let warning = null;
+  let warning: string | null = null;
```

Comment:
```suggestion
  let warning?: string = null;
```
Seen this in a few PRs now is there a reason to use `string | null` instead?

Reply:
`let warning?: string = null;`

If you do this, `?` means the string can only be a value or `undefined`. You have to add `null` as an additional permitted type, OR initialise the value to be `undefined` which is not as nice as initialising it to `null`

Reply:
thanks, nice thing to learn here, I'm all for the distinction between null and undefined